core/src/main/java/hudson/Launcher.java

```diff
@@ -181,6 +181,8 @@ public Computer getComputer() {
          */
         protected boolean reverseStdin, reverseStdout, reverseStderr;
 
+        protected boolean dontKillWhenInterrupted;
```

Comment:
Some documentation would be useful + `@since TODO`

Reply:
Done